shell/browser/atom_browser_client.cc

```diff
@@ -979,13 +980,14 @@ bool AtomBrowserClient::WillCreateURLLoaderFactory(
     bool* bypass_redirect_checks) {
   content::WebContents* web_contents =
       content::WebContents::FromRenderFrameHost(frame_host);
-  if (!web_contents) {
```

Comment:
This was added for service workers that crashed here since they don't have any associated `web_contents`. https://github.com/electron/electron/pull/19488